src/groupeddataframe/utils.jl

```diff
@@ -5,7 +5,7 @@ function hashrows_col!(h::Vector{UInt},
                        v::AbstractVector{T},
                        rp::Nothing,
                        firstcol::Bool) where T
-    @spawn_for_chunks 1_000_000 for i in eachindex(h)
+    Threads.@threads for i in eachindex(h)
```

Comment:
Are you sure it's a good idea to use `@threads` here? For simple cases like this, it sounds better to use `@spawn` as it can work even with nested parallelism, and the overhead is low.